docs/team/teerenjing.md

```diff
@@ -9,22 +9,27 @@ BayMeds v.2103 is a desktop application used for tracking medications. The user
 
 Given below are my contributions to the project.
 
+* **New Feature**: Added the ability to add conflicting drugs.
+  * What it does: Allows the user to add multiple conflicting drugs to a prescription.
+  * Justification: This feature allows a user to specify which drugs are conflicting so as to manage the possible clashes when planning the consumption schedule.
+  * Highlights: This enhancement affects existing commands and commands to be added in future.
+    The implementation was also complex as the algorithm ensures that the conflicting relationship between drugs are bidirectional.
+    If drug A conflicts with drug B, the algorithm ensures that drug B must also conflict with drug A.  
+  * Future plans: Seperate drug names by commas instead of spaces to allow for more complex drug names that might have multiple words. 
+  
```

Comment:
Engrish `seperate`.

Also why a new section rather than under enhancements? Is thr a reason.

Reply:
I was thinking that conflicting drugs feature was not in the original ab3. Hence, I put it in a new section.